src/solve/test.rs

```diff
@@ -1680,3 +1680,36 @@ fn partial_overlap_2() {
         }
     }
 }
+
+#[test]
+#[ignore]
```

Comment:
I'd prefer not to have `#[ignore]` but rather include the current result that we see, with a comment that it is a bug. (I don't like to ignore tests, personally, it's hard to remember when to un-ignore them.)